FlowCrypt/Controllers/ImportKey/ImportKeyViewController.swift

```diff
@@ -0,0 +1,207 @@
+        navigationController?.setNavigationBarHidden(false, animated: animated)
+        navigationController?.navigationBar.barStyle = .black
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        userInfoMessage = ""
+    }
+
+    private func setupUI() {
+        node.delegate = self
+        node.dataSource = self
+        title = decorator.sceneTitle
+    }
+
+    private func updateSubtitle() {
+        DispatchQueue.main.async {
+            self.node.reloadRows(at: [Parts.description.indexPath], with: .fade)
+        }
+    } 
+}
+
+// MARK: - ASTableDelegate, ASTableDataSource
+
+extension ImportKeyViewController: ASTableDelegate, ASTableDataSource {
+    func tableNode(_ tableNode: ASTableNode, numberOfRowsInSection section: Int) -> Int {
+        return Parts.allCases.count
+    }
+
+    func tableNode(_ tableNode: ASTableNode, nodeBlockForRowAt indexPath: IndexPath) -> ASCellNodeBlock {
+        return { [weak self] in
+            guard let self = self, let part = Parts(rawValue: indexPath.row) else { return ASCellNode() }
+            switch part {
+            case .title:
+                return SetupTitleNode(
+                    title: self.decorator.title,
+                    insets: self.decorator.titleInsets
+                )
+            case .description:
+                return SetupTitleNode(
+                    title: self.decorator.subtitleStyle(self.userInfoMessage),
+                    insets: self.decorator.subTitleInset
+                )
+            case .fileImport:
+                return SetupButtonNode(
+                    title: self.decorator.fileImportTitle,
+                    insets: self.decorator.buttonInsets
+                ) { [weak self] in
+                    self?.proceedToKeyImportFromFile()
+                }
+            case .pasteBoardImport:
+                return SetupButtonNode(
+                    title: self.decorator.pasteBoardTitle,
+                    insets: self.decorator.buttonInsets
+                ) { [weak self] in
+                    self?.proceedToKeyImportFromPasteboard()
+                }
+                .then {
+                    $0.isButtonEnabled = self.pasteboard.hasStrings
+                }
+            }
+        }
+    }
+}
+
+// MARK: - Actions
+
+extension ImportKeyViewController {
+    private func proceedToKeyImportFromFile() {
+        let acceptableDocumentTypes = [
+            String(kUTTypeText),
+            String(kUTTypePlainText),
+            String(kUTTypeUTF8PlainText),
+            String(kUTTypeUTF16ExternalPlainText),
+            String(kUTTypeUTF16PlainText),
+            String(kUTTypeDelimitedText),
+            String(kUTTypeCommaSeparatedText),
+            String(kUTTypeTabSeparatedText),
+            String(kUTTypeUTF8TabSeparatedText),
+            String(kUTTypeRTF),
+            String(kUTTypePDF),
+            String(kUTTypeItem)
```

Comment:
Maybe add `kUTTypeData`? Sometimes keys can be binary.

Also would remove PDF, RTF, kUTTypeUTF8TabSeparatedText, kUTTypeTabSeparatedText, kUTTypeDelimitedText, kUTTypeCommaSeparatedText

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/142